### PR TITLE
SymptomCard and SymptomsList components also antd added

### PR DESCRIPTION
**Pull request by JuanYut**
SymptomCard and SymptomsList components also antd added

### CODE REVIEW

File: src/styles/pages/symptomsList.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  &__cards {
+    padding: 60px 0 60px 0;
+    width: 70%;
+    background-color: #dfdfdf;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    border-radius: 10px;
+  }
+}
```
Comment (JuanYut) [inline src/styles/pages/symptomsList.scss:18]: Example of how to use BEM in CSS or SCSS 👻

**Review by xErikPsx — APPROVED** [review]
Nice.

File: src/components/Pages/Symptoms/SymptomsList.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+function SymptompsList() {
+  return (
+    <div className="symptomsList">
+      <h2>Síntomas del Covid-19</h2>
+      <div className="symptomsList__cards">
+        <SymptomCard
+          title={"Title"}
+          description={"Lorem algo algo asjdahsdiawd"}
+          image={"https://os.alipayobjects.com/rmsportal/QBnOOoLaAfKPirc.png"}
+        />
```
Comment (JuanYut) [inline src/components/Pages/Symptoms/SymptomsList.jsx:14]: In the List of Cards for "Symptoms" we just need to import the card component, ez game 👻

File: src/containers/Symptoms/Symptoms.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+function Symptomps() {
+  return (
+    <div>
+      <SymptomsList />
+    </div>
+  );
+}
+
+export default Symptomps;
```
Comment (JuanYut) [inline src/containers/Symptoms/Symptoms.jsx:13]: In the container for Symptoms, we just need to import and use the List of Symptoms 👻

Comment (aherreDev) [reply src/containers/Symptoms/Symptoms.jsx:13]: Great!

**Pull request merged**